swan-cern/templates/fluentd/fluentd_sources.conf.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-sources-conf
+  namespace: {{ .Release.namespace }}
+data:
+  sources.conf: |-
+      # Do not collect fluentd logs to avoid infinite loops.
+      <match kubernetes.var.log.containers.**fluentd**>
+          @type null
+      </match>
+
+    {{ if not .Values.fluentd.output.includeInternal }}
```

Comment:
iirc this was present in the old fluentd chart, and we are now copying it here? And this is to discard the logs of all the kube-system namespace containers?

Reply:
Yes, it was present in the old upstream chart and was copied here since it is not present in the official one. It is to discard the logs of all the kube-system namespace containers.